server/services/cohorts.js

```diff
@@ -75,9 +75,17 @@ const getPSICohorts = async (psiID) =>
           cohort_id: 'id',
         },
       },
+      participantStatusJoin: {
+        type: 'LEFT OUTER',
+        relation: collections.PARTICIPANTS_STATUS,
+        on: {
+          participant_id: 'participants.participant_id',
+        },
+      },
     })
     .find({
       psi_id: psiID,
+      'participantStatusJoin.current !=': false,
```

Comment:
Why the double negative here instead of just `true`?

Reply:
Is there more criteria we can go off of? I'm wondering if we should be expecting certain statuses for cohort participants. What status should cohort participants be? Just checking for statuses that are current doesn't seem straight forward

Reply:
When setting `'participantStatusJoin.current': true,` it'll select all cohorts that have participants with statuses set to `true` and omit all the cohorts that don't have any participants at all. So, instead of doing `or: ['participantStatusJoin.current': true, 'participantStatusJoin.current': null]` I'm checking that it's not `false`.
Also, we're not looking for any particular statuses, just checking that participants have some statuses assigned to them. Alternatively, we can check is the status is `hired` or `archived`

Reply:
I guess there aren't any cases where they wouldn't be hired or archived at this point right? This is definitely a tough piece to understand

Which participants are we trying to omit by adding this condition?

Reply:
But how would a participant assigned to a cohort not have any participant statuses?

Reply:
I suspect it might be some legacy data in dev and test

Reply:
I hesitate to change application code based on data that we don't expect to exist, @fw-noel how do you feel about this? As an alternative we could not fix this here but remove the problematic records in dev/test

At the very least I'd advocate for adding a comment that this is an additional guard to protect against invalid records

Reply:
Adding a comment makes total sense to me 👍
What about check for `hired` and `archived` participants? I feel like it might be a good idea to add it, since we're already using it in the reports